api/chat_main_handler.go

```diff
@@ -176,22 +171,6 @@ func (h *ChatHandler) OpenAIChatCompletionAPIWithStreamHandler(w http.ResponseWr
 		return
 	}
 
-	////
-
-	// no session exists
-	//
-	// if no session chat_created, create new chat_session with $uuid
-	// create a new prompt with topic = $uuid, role = "system", content= req.Prompt
-
-	// if session avaiable,
-	// GetChatPromptBySessionID and create Message from Prompt
-	// GetLatestMessagesBySessionID  and create Messsage(s) from messages
-
-	// Check if the chat session exists
-
-	// no session exists
-	// create session and prompt
-
 	chatSession, err := h.chatService.q.GetChatSessionByUUID(ctx, chatSessionUuid)
 
 	if err != nil {
```

Comment:
:robot: OpenAI

This patch removes some commented-out code that is no longer needed.

This change does not introduce any new functionality or bug risks.

Please confirm that the removed code is not needed before merging this patch.

Also, please update the comments on the pull request to reflect this change.

<!-- This is an auto-generated comment by OpenAI -->